scripts/exgfs_wave_post_pnt.sh

```diff
@@ -156,7 +156,7 @@ source "${USHgfs}/preamble.sh"
     cp -f ${PARMgfs}/wave/wave_${NET}.buoys buoy.loc.temp
     if [ "$DOBNDPNT_WAV" = YES ]; then
       #only do boundary points
-      sed -n '/^\$.*/!p' buoy.loc.temp | grep IBP > buoy.loc || {
+      sed -n '/^\$.*IBP.*/!p' buoy.loc.temp > buoy.loc || {
```

Comment:
Can we split the buoy lists into boundary and not and avoid this step entirely? I don't think there's any reason that this filtering needs to be done every time.